### PR TITLE
Task Gruzinova

### PR DESCRIPTION
**Pull request by Marymaryyyyyy**

### CODE REVIEW

File: Class1.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Task_dev_1
+{
+    /// <summary>
+    /// This class is looking for repeating symbols in a string.
+    /// </summary>
+  
+   class repeatingSymbols
```
Comment (AlexanderSoroka) [inline Class1.cs]: useless blank line

File: Class1.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  
+   class repeatingSymbols
+    {
+        private string inputString;
+
+        public repeatingSymbols(string args)
+        {
+            inputString = args;
+        }
+        /// <summary>
```
Comment (AlexanderSoroka) [inline Class1.cs]: add blank line prior comment

File: Class1.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                }
+                else
+                {
+                    if (count > maxCount)
+                    {
+                        maxCount = count;
+                    }
+                    count = 1;
+                }
+
```
Comment (AlexanderSoroka) [inline Class1.cs]: useless blank line

File: Class1.cs [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            }
+            return maxCount;
+
```
Comment (AlexanderSoroka) [inline Class1.cs]: useless blank line

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: Class1.cs [hunk 5]
```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace Task_dev_1
+{
+    /// <summary>
+    /// This class is looking for repeating symbols in a string.
+    /// </summary>
+  
+    class repeatingSymbols
```
Comment (IgorKheidorov) [inline Class1.cs]: Code style, upper case

File: Class1.cs [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+        public repeatingSymbols(string args)
+        {
+            inputString = args;
+        }
+        /// <summary>
+        /// This method finds the maximum number of identical symbols in a string
+        /// </summary>
+        /// <returns>This method prints the maximum number of identical symbols in the string</returns>
+        public int MaxNumberrepeatingSymbols()
```
Comment (IgorKheidorov) [inline Class1.cs]: better MaxNumberRepeatingSymbol

File: Class1.cs [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+                {
+                    if (count > maxCount)
+                    {
+                        maxCount = count;
+                    }
+                    count = 1;
+                }
+             }
+            return maxCount;
+
```
Comment (IgorKheidorov) [inline Class1.cs]: Code style, empty line

File: Task dev_1.cs [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        /// </summary>
+        /// <param name="args">String to input</param>
+        static void Main(string[] args)
+        {
+            repeatingSymbols inputString = new repeatingSymbols(args[0]);
+            Console.WriteLine(inputString.MaxNumberrepeatingSymbols());
+            Console.ReadKey();
+        }
+    }
+
```
Comment (IgorKheidorov) [inline Task dev_1.cs]: Empty string

File: Class1.cs [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+        {
+            inputString = args;
+        }
+        /// <summary>
+        /// This method finds the maximum number of identical symbols in a string
+        /// </summary>
+        /// <returns>This method prints the maximum number of identical symbols in the string</returns>
+        public int MaxNumberrepeatingSymbols()
+        {
+            int count = 1;
```
Comment (IgorKheidorov) [inline Class1.cs]: Better currentCount

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Add other necessary files to request(solution, proj, etc)

File: Class1.cs [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+        private string inputString;
+
+        public RepeatingSymbols(string args)
+        {
+            inputString = args;
+        }
+        /// <summary>
+        /// This method finds the maximum number of identical symbols in the string
+        /// </summary>
+        /// <returns>This method prints the maximum number of identical symbols in the string</returns>
```
Comment (IgorKheidorov) [inline Class1.cs]: Bad comment, this method prints nothing, its returns number, rewrite comment

File: Class1.cs [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+        public int MaxNumberRepeatingSymbols()
+        {
+            int currentCount = 1;
+            int maxCount = 1;
+
+            for (int i = 0; i < inputString.Length - 1; i++)
```
Comment (IgorKheidorov) [inline Class1.cs:26]: Bug

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: Class1.cs [hunk 12]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
```
Comment (AlexanderSoroka) [inline Class1.cs:1]: Inappropriate filename

**Review by AlexanderSoroka — APPROVED** [review]
Not accepted as some comments were just ignored

**Pull request merged**